model/execution/implementation/Lti1p3ContextCacheRepository.php

```diff
@@ -38,6 +38,18 @@ public function __construct(?CacheItemPoolInterface $cache = null)
         $this->cache = $cache;
     }
 
+    public function findByDeliveryExecutionId($deliveryExecutionId): ?LtiLaunchData
```

Comment:
Let's cover it as well in a test: https://github.com/oat-sa/extension-tao-ltideliveryprovider/blob/develop/test/unit/model/Lti1p3ContextCacheRepositoryTest.php#L59

Reply:
Done